contracts/PolygonZkEVM.sol

```diff
@@ -580,6 +580,9 @@ contract PolygonZkEVM is
         // Consolidate pending state if possible
         _tryConsolidatePendingState();
 
+        // Update global exit root if there's new deposits
```

Comment:
```suggestion
        // Update global exit root if there are new deposits
```

Reply:
This will be always triggered when sequencing.
I would remove comments or just add the logic to update global exit root when there are new deposits

Reply:
ok...this is handled inside the Bridge xD